src/Diagnostics.RuntimeHost/Services/ResourceService/StampService.cs

```diff
@@ -46,15 +46,16 @@ public StampService()
 
             var windowsTenantIdsTask = GetTenantIdsAsync(stamp, startTime, endTime, dataProviderContext, PlatformType.Windows);
             var linuxTenantIdsTask = GetTenantIdsAsync(stamp, startTime, endTime, dataProviderContext, PlatformType.Linux);
+            await Task.WhenAll(windowsTenantIdsTask, linuxTenantIdsTask);
```

Comment:
You dont have to do this since they are awaited in the below 2 lines of code

Reply:
That could result in `UnobservedTaskException` if the first `Task` throws.
https://stackoverflow.com/a/49952821/3524426

Reply:
Oh this is interesting, I haven't heard of this before. I was learning a little about it and I think you should be fine actually. Apparently as long as you have a reference to the task and you are not doing a fire and forget then you should be fine. 
https://stackoverflow.com/questions/3284137/taskscheduler-unobservedtaskexception-event-handler-never-being-triggered

Reply:
I can't reproduce it in .NET Core actually, so it's probably not an issue anymore